libycp/src/YCPBuiltinString.cc

```diff
@@ -299,28 +299,28 @@ s_substring1 (const YCPString &s, const YCPInteger &i1)
     if (s->isAscii())
     {
 	string ss = s->value ();
-	string::size_type start = i1->value();
+	long long start = i1->value();
```

Comment:
Doesn't it break trick with having i1 -1 for max? I just want to be sure we do not break anything.